src/MySql/readme.md

```diff
@@ -47,31 +47,44 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
-branch: ae862b1c090b4c2c951ea46bf97ddbafd6f76d82
+branch: ea6d1725ca9669714cd5f5f969d026b90ecffbd1
 require:
   - $(this-folder)/../readme.azure.noprofile.md
 input-file:
   - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json
   - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/ServerSecurityAlertPolicies.json
+  - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2020-07-01-preview/mysql.json
 module-version: 0.1.0
 title: MySQL
 subject-prefix: 'MySQL'
 
 directive:
-  - from: mysql.json
```

Comment:
We need PG flexible server as well..! Could you do the same generation for postgresql as well? 

Single server"input-file:
- Microsoft.DBforPostgreSQL/stable/2017-12-01/postgresql.json
- Microsoft.DBforPostgreSQL/stable/2017-12-01/ServerSecurityAlertPolicies.json
- Microsoft.DBforPostgreSQL/stable/2018-06-01/PrivateEndpointConnections.json
- Microsoft.DBforPostgreSQL/stable/2018-06-01/PrivateLinkResources.json
- Microsoft.DBforPostgreSQL/stable/2020-01-01/DataEncryptionKeys.json

flexible server
input-file:
- Microsoft.DBforPostgreSQL/preview/2020-02-14-preview/postgresql.json

Reply:
Yes, we will do the same thing for postgresql. But let's firstly try everything in mysql and ensure them to work and then we are migrate them to postgresql quickly.